client/my-sites/customer-home/cards/education/free-photo-library/index.jsx

```diff
@@ -3,76 +3,42 @@
  */
 import React from 'react';
 import { useTranslate } from 'i18n-calypso';
-import { Button, Card } from '@automattic/components';
-import { isDesktop } from '@automattic/viewport';
-import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import CardHeading from 'components/card-heading';
-import { openSupportArticleDialog } from 'state/inline-support-article/actions';
 import { localizeUrl } from 'lib/i18n-utils';
-import {
-	withAnalytics,
-	composeAnalytics,
-	recordTracksEvent,
-	bumpStat,
-} from 'state/analytics/actions';
-
-/**
- * Style dependencies
- */
-import './style.scss';
+import InlineSupportLink from 'components/inline-support-link';
+import EducationalContent from '../educational-content';
 
 /**
  * Image dependencies
  */
 import freePhotoLibraryVideoPrompt from 'assets/images/customer-home/illustration--free-photo-library.svg';
 
-const FreePhotoLibrary = ( { openSupportArticleDialogAndTrack } ) => {
+const FreePhotoLibrary = () => {
 	const translate = useTranslate();
 
 	return (
-		<Card className="free-photo-library">
-			{ isDesktop() && (
-				<button onClick={ openSupportArticleDialogAndTrack.bind( this, 'prompt' ) }>
-					<img
-						className="free-photo-library__demonstration-image"
-						src={ freePhotoLibraryVideoPrompt }
-						alt={ translate( 'Free Photo Library demonstration' ) }
-					/>
-				</button>
+		<EducationalContent
+			title={ translate( 'The WordPress.com free photo library' ) }
+			description={ translate(
+				'Our free photo library integrates your site with over 40,000 beautiful copyright-free photos to create stunning designs.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'The WordPress.com Free Photo Library integrates your site with beautiful copyright-free photos to create stunning designs.' )` *ES Score: 8*